### DISCUSSION TITLE
There's a problem with my billing information

### DISCUSSION THREAD

**Discussion by baiwandanjiwang** [community/community#147864] [Other Feature Feedback, Questions, & Ideas]
![屏幕截图 2024-12-18 144952](https://github.com/user-attachments/assets/d8be4542-b6d4-41ed-8008-c08d13da62e7)
### Select Topic Area

Question

### Body

My region is set in China, which is not in those restricted regions, but you still restrict me and don't let me change my billing information. Please help me undo the restrictions, otherwise I can't use the paid items.

**Comment by davevad93** [comment]
Hi @baiwandanjiwang ,

You should contact support by opening a ticket [here](https://support.github.com/contact/account), select "`Billing, payments, or receipts`" and explain your issue.
